_rules/audio-as-media-alternative-afb423.md

```diff
@@ -3,8 +3,8 @@ id: afb423
 name: Audio-only as a media alternative for text
 rule_type: atomic
 description: |
-  This rule checks `audio` is a media alternative for text on the page.
-test_aspects:
+  This rule checks `audio` is a media alternative for text on the page. 
+input_aspects:
```

Comment:
We need to have the accessibility_requirements section here, even though it might will be empty :)